routes.py

```diff
@@ -1,91 +0,0 @@
-#  Created by Bogdan Trif on 2022.05.25 , 4:44 PM ; btrif
-from flask import request, render_template
-
-from settings import *
-
-
-####         Routes         #####
-@app.route('/')
-def index():
-    return render_template('index.html' )
-    return {"status": "success" , "message" : "This is the Parking Car Flask API" }
```

Comment:
There are 2 return statements for a single function

Reply:
the last one will NEVER be executed. Can be removed.

Reply:
completed. in Oracle_review 2